### PR TITLE
Adapted IAM/ADMIN communication

### PR DESCRIPTION
**Pull request by umbreak**
Fixes https://github.com/BlueBrain/nexus/issues/1218

### CODE REVIEW

File: service/src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationCommand.scala [hunk 1]
```diff
@@ -3,7 +3,8 @@ package ch.epfl.bluebrain.nexus.admin.organizations
 import java.time.Instant
 import java.util.UUID
 
-import ch.epfl.bluebrain.nexus.iam.client.types.Identity.Subject
+import ch.epfl.bluebrain.nexus.iam.types.Identity.Subject
+import ch.epfl.bluebrain.nexus.iam.types.Identity.Subject
```
Comment (bogdanromanx) [inline service/src/main/scala/ch/epfl/bluebrain/nexus/admin/organizations/OrganizationCommand.scala:7]: duplicate import

File: service/src/test/scala/ch/epfl/bluebrain/nexus/admin/routes/ProjectRoutesSpec.scala [hunk 2]
```diff
@@ -178,8 +209,7 @@ class ProjectRoutesSpec
     }
 
     "create a project without optional fields" in new Context {
-      iamClient.hasPermission(Path("/org").rightValue, create)(any[Option[AuthToken]]) shouldReturn Task.pure(true)
-      iamClient.identities shouldReturn Task(caller)
+//      iamClient.hasPermission(Path("/org").rightValue, create)(any[Option[AuthToken]]) shouldReturn Task.pure(true)
```
Comment (bogdanromanx) [inline service/src/test/scala/ch/epfl/bluebrain/nexus/admin/routes/ProjectRoutesSpec.scala:212]: why do we need to keep these commented lines?

File: service/src/main/scala/ch/epfl/bluebrain/nexus/service/exceptions/ServiceError.scala [hunk 3]
```diff
@@ -36,7 +36,6 @@ object ServiceError {
     case i: IamError   => IamError.iamErrorEncoder(i)
     case a: AdminError => AdminError.adminErrorEncoder(a)
     case r =>
-      println("dupa dupa")
```
Comment (bogdanromanx) [inline service/src/main/scala/ch/epfl/bluebrain/nexus/service/exceptions/ServiceError.scala:39]: nice catch :)

Comment (wwajerowicz) [reply service/src/main/scala/ch/epfl/bluebrain/nexus/service/exceptions/ServiceError.scala:39]: oops :)

File: service/src/main/scala/ch/epfl/bluebrain/nexus/admin/config/AdminConfig.scala [hunk 4]
```diff
@@ -32,7 +30,7 @@ final case class AdminConfig(
 object AdminConfig {
 
   final case class ServiceAccountConfig(token: Option[String]) {
-    def credentials: Option[AuthToken] = token.map(AuthToken)
+    def credentials: Option[AccessToken] = token.map(AccessToken)
```
Comment (wwajerowicz) [inline service/src/main/scala/ch/epfl/bluebrain/nexus/admin/config/AdminConfig.scala:33]: It's a bit weird  to be  HTTP Bearer token when not doing interacting with a service via HTTP.  We can probably refactor that later.

Comment (umbreak) [reply service/src/main/scala/ch/epfl/bluebrain/nexus/admin/config/AdminConfig.scala:33]: Yeah. The thing is we still a `Subject` to make calls to the `Realms` and `Acls` API.
We could have instead something like
```scala
final case class ServiceAccount(realm: String, name: String){
   def subject: Subject = User(realm, name)
}
```

But that's more prone to errors than having the actual saToken from where the subject is going to be extracted by the JWT automatically

**Pull request merged**